### ISSUE TITLE
Pytorch-UNet/unet/unet_parts.py， padding problem

### ISSUE THREAD

**Issue by guoruoqian** [milesial/Pytorch-UNet#4]
In http://pytorch.org/docs/0.3.0/nn.html?highlight=pad#torch.nn.functional.pad    , "4D input tensor pad should be in form (pad_l, pad_r, pad_t, pad_b )".
but in your code,
https://github.com/milesial/Pytorch-UNet/blob/2c17549a3dc926730e9fcd16ec18610a9e5ec391/unet/unet_parts.py#L68

**Comment by milesial** [comment]
The function call is split on two lines, take a look at the next line :wink: 
https://github.com/milesial/Pytorch-UNet/blob/2c17549a3dc926730e9fcd16ec18610a9e5ec391/unet/unet_parts.py#L68-L69
